### PR TITLE
Fix deprecated warnings

### PR DESCRIPTION
**Pull request by ncihnegn**
Like https://github.com/diml/lambda-term/pull/63

### CODE REVIEW

File: utop.opam [hunk 1]
```diff
@@ -22,4 +22,4 @@ depends: [
   "jbuilder"     {build & >= "1.0+beta9"}
 ]
 build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
-available: [ocaml-version >= "4.02.3"]
+available: [ocaml-version >= "4.03.0"]
```
Comment (diml) [inline utop.opam]: Why this change?

Comment (ncihnegn) [reply utop.opam]: String.lowercase_ascii/captialize_ascii are since 4.03.0

**Comment by XVilka** [comment]
So what about this PR? Seems it is good enough to be merged.

**Comment by diml** [comment]
The travis file needs to be updated as 4.02 is no longer supported after this PR

**Comment by pmetzger** [comment]
We could separately commit a fix to the travis file and then merge this...

**Comment by XVilka** [comment]
@pmetzger @diml see https://github.com/ocaml-community/utop/pull/254

**Comment by diml** [comment]
@ncihnegn, could you rebase your branch?

**Comment by diml** [comment]
Meta comment: on some other projects I work on, I can see a button "Update branch", which allows me as a maintainer to merge master in the PR. This is convenient, I often do that to make sure the CI passes and then do "Rebase and merge" or "Squash and merge". However, I don't see this button here, not sure why.

**Comment by pmetzger** [comment]
I've never seen that button on other projects. It might be a non-default setting we need to turn on.

**Comment by pmetzger** [comment]
It looks like the only failures were on 4.02, so should we rebase and merge?

**Comment by whitequark** [comment]
@diml Click the small down arrow near the button, it lets you select other merge typse.

**Comment by whitequark** [comment]
@diml Sorry, I misread, disregard my earlier comment.

**Comment by diml** [comment]
> I've never seen that button on other projects. It might be a non-default setting we need to turn on.

Possibly. BTW, I don't seem to be able to access utop's settings, would it be possible to add me as an admin for utop?

> It looks like the only failures were on 4.02, so should we rebase and merge?

The other PR also added 4.07, so we don't know yet if this PR succeeds on 4.07.

**Comment by pmetzger** [comment]
> would it be possible to add me as an admin for utop?

Sure, but I can't find such a setting. Tell me what to click. :)

**Comment by diml** [comment]
If you go in settings, then "outside collaborators", you can add me as an admin

**Comment by pmetzger** [comment]
Done.

**Comment by XVilka** [comment]
Tired of waiting and did rebase myself: https://github.com/ocaml-community/utop/pull/255
